ci/pkl-pipelines/pay-dev/e2e-helpers.pkl

```diff
@@ -8,6 +8,34 @@ import "../common/shared_resources_for_test_pipelines.pkl" as shared_test
 import "../common/shared_resources_for_slack_notifications.pkl"
 import "../common/PayResources.pkl"
 
+local class ImageToCopyFromDockerhubToECR {
+  name: String
+  repo: String
+  tag: String
+  image: String = "\(repo):\(tag)"
+}
+
+local images_to_copy_from_dockerhub_to_ecr = new Listing<ImageToCopyFromDockerhubToECR> {
+  new { name = "postgres-15-alpine" repo = "postgres" tag = "15-alpine" }
+  new { name = "localstack-localstack-3" repo = "localstack/localstack" tag = "3" }
+  new { name = "selenium-standalone-chrome-3-141-59" repo = "selenium/standalone-chrome" tag = "3.141.59" }
+}
+
+local class ImageToMultiArchBuild {
+  name: String
+  github_repo: String
+  dockerhub_repo: String = "governmentdigitalservice/pay-\(name)"
+  ecr_repo: String = "govukpay/\(name)"
```

Comment:
Just reminded how nifty this interpolation syntax is - good ol' pkl 😻

Reply:
Yeah the late binding makes it incredible! It makes me really happy :)